internal/controller/promise_controller.go

```diff
@@ -316,12 +317,34 @@ func (r *PromiseReconciler) nextReconciliation(promise *v1alpha1.Promise, logger
 func (r *PromiseReconciler) setPromiseStatusToAvailable(ctx context.Context, promise *v1alpha1.Promise, logger logr.Logger) (ctrl.Result, error) {
 	logger.Info("Promise status being set to Available")
 	promise.Status.Status = v1alpha1.PromiseStatusAvailable
-	promise.Status.LastAvailableTime = &metav1.Time{Time: time.Now()}
+	timestamp := metav1.Time{Time: time.Now()}
+	promise.Status.LastAvailableTime = &timestamp
+	updateConditionOnPromise(promise, promiseAvailableStatusCondition(timestamp))
 
 	r.EventRecorder.Eventf(promise, "Normal", "Available", "Promise is available")
 	return r.updatePromiseStatus(ctx, promise)
 }
 
+func promiseAvailableStatusCondition(lastTransitionTime metav1.Time) metav1.Condition {
+	return metav1.Condition{
+		Type:               v1alpha1.PromiseAvailableConditionType,
+		LastTransitionTime: lastTransitionTime,
+		Status:             metav1.ConditionTrue,
+		Message:            "Ready to accept resource requests",
```

Comment:
```suggestion
		Message:            "Ready to fulfil resource requests",
```

Resource requests will still be accepted by k8s, but not reconciled on (fulfilled) when Promises aren't available